OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -110,3 +114,65 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
 ## ORA-01157: cannot identify/lock data file
 
 This error occurs when the database cannot find a data file (used for tablespaces) that was previously present. This is most likely because the data file has been located outside the volume in a previous container and was hence not persisted. Ensure that when you add tablespaces and/or data files that they are located within the volume location, i.e. $ORACLE_BASE/oradata/$ORACLE_SID, (e.g. `/opt/oracle/oradata/XE`).
+
+## Running Oracle Database 23c Free on Apple based ARM M1 chip
```

Comment:
This probably applies to M2 as well. Perhaps use a less version-specific title?  Apple Silicon?  Apple ARM?  IDK.